Migrate/alert list

Chanputhi opened the pull request:
_Any italic text should be deleted from the final Pull Request text, including this line_

# Description

_Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change._

Fixes # (issue)

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

_Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration_

## Testing Checklist:

- [ ] Tested in latest Chrome
- [ ] Tested in latest Safari
- [ ] Tested in latest Firefox

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have requested a review from @macite and @jakerenzella on the Pull Request

Chanputhi committed:
build: create initial files for alert-list migration

Chanputhi committed:
test: migrate alert list

Chanputhi commented:
I have completed the code for this component already. However, this component is no longer used in the future because of the new branch using snackBar. Therefore, I will be moving to the next component. I have tried to solve it for a few days but it is not working because The alert variable in Angular JS cannot move to the root scope of Angular. Therefore, this component will no longer be used.

Chanputhi committed:
migrate: alert-list

Chanputhi committed:
Migrate: Make some change for alert-list component

lst97 commented on src/app/common/alert-list/alert-list.component.html:
```diff
@@ -0,0 +1,25 @@
+
+<div class="alert-top">
+  {{alertService.alerts}} Test
+    <alert *ngFor="let alert of alertService.alerts" [type]="alert.type" [close]="alert.close()">
```
@Chanputhi I think the migration may not be fully satisfy, because the alerts is from the rootScope in AngularJS and it is not accessible from Angular, it may raise error, please double check that. Thank you.

Please change this pull request to draft.

Chanputhi commented:
This is the old version of OnTrack which I have implemented before the new version released. I have moved it to the new version already. Here is the link: https://github.com/doubtfire-lms/doubtfire-web/pull/665

lst97 commented:
Covert it to draft or close it if duplicated, thank you.
